MekHQ/src/mekhq/Version.java

```diff
@@ -85,86 +89,105 @@ public int getMajorVersion() {
-    	return versionCompare(new Version(checkVersion));
+    public boolean isHigherThan(String other) {
+        return isHigherThan(new Version(other));
     }
-    
+
     /**
      * Compare Versions
-     * 
+     *
      * Use this method to determine if the version passed is less than
      * this Version object.
-     * 
-     * @param checkVersion  The version we want to see if is less than this version
-     * 
+     *
+     * @param other  The version we want to see if is less than this version
+     *
      * @return true if checkVersion is less than this Version object
      */
-    public boolean versionCompare(Version checkVersion) {
-    	// Pass to the static method for the final computation
-    	return Version.versionCompare(checkVersion, this);
-    }
-    
-    /**
-     * Compare Versions
-     * 
-     * @param checkVersion  The version we want to see if is less than the other
-     * @param staticVersion The static version that we're comparing against
-     * 
-     * @return true if checkVersion is less than staticVersion
-     */
-    public static boolean versionCompare(String checkVersion, String staticVersion) {
-    	return versionCompare(new Version(checkVersion), new Version(staticVersion));
+    public boolean isHigherThan(Version other) {
+        return Version.compare(this, other) > 0;
     }
-    
+
     /**
      * Compare Versions
-     * 
-     * @param checkVersion  The version we want to see if is less than the other
-     * @param staticVersion The static version that we're comparing against
-     * 
-     * @return true if checkVersion is less than staticVersion
+     *
+     * Use this method to determine if this version is lower than
+     * the version passed
+     *
+     * @param other The version we want to see if it is higher than this version.
+     *
+     * @return true if this is lower than checkVersion
      */
-    public static boolean versionCompare(Version checkVersion, String staticVersion) {
-    	return versionCompare(checkVersion, new Version(staticVersion));
+    public boolean isLowerThan(String other) {
+        return isLowerThan(new Version(other));
     }
-    
+
     /**
      * Compare Versions
-     * 
-     * @param checkVersion  The version we want to see if is less than the other
-     * @param staticVersion The static version that we're comparing against
-     * 
-     * @return true if checkVersion is less than staticVersion
+     *
+     * Use this method to determine if this version is lower than
+     * the version passed
+     *
+     * @param other The version we want to see if it is higher than this version.
+     *
+     * @return true if this is lower than checkVersion
      */
-    public static boolean versionCompare(String checkVersion, Version staticVersion) {
-    	return versionCompare(new Version(checkVersion), staticVersion);
+    public boolean isLowerThan(Version other) {
+            return Version.compare(other, this) > 0;
```

Comment:
prefer the symmetry of `Version.compare(this, other) < 0`

Reply:
Changed :)